mne/time_frequency/tfr.py

```diff
@@ -687,6 +687,72 @@ def tfr_morlet(inst, freqs, n_cycles, use_fft=False, return_itc=True, decim=1,
 
 
 @verbose
+def tfr_array_morlet(epoch_data, frequencies, sfreq=1.0, n_cycles=7.0,
+                     zero_mean=False, use_fft=True, decim=1, output='complex',
+                     n_jobs=1, verbose=None):
+    """Compute time-frequency transforms.
+
+    Convolves a Morlet wavelet.
```

Comment:
I'd be a little more specific like "Convolves epoch data with selected Morlet wavelets"